esp/esp/users/models/__init__.py

```diff
@@ -1354,6 +1362,7 @@ class TeacherInfo(models.Model, CustomFormsLinkModel):
 
     user = AjaxForeignKey(ESPUser, blank=True, null=True)
     graduation_year = models.CharField(max_length=4, blank=True, null=True)
+    affiliation = models.CharField(max_length=100, blank=True, null=True)
     from_here = models.NullBooleanField(null=True)
```

Comment:
Hmm, thoughts on removing the old fields and just making them `@property`s?  (This will require some more migrations.)